test/gui/shared/steps/account_context.py

```diff
@@ -196,3 +196,22 @@ def step(context, action):
     else:
         AccountConnectionWizard.cancelEnableExperimentalVFSOption()
     AccountConnectionWizard.nextStep()
+
+
+@When('the user adds the following account information:')
+def step(context):
+    print(context)
+    account_details = getClientDetails(context)
+    print(account_details['server'])
+    AccountConnectionWizard.addServer(account_details['server'])
+    if get_config('ocis'):
+        AccountConnectionWizard.acceptCertificate()
+    AccountConnectionWizard.addUserCreds(
+        account_details['user'], account_details['password']
+    )
+
+
+@Then("local folder selection option should be visible")
+def step(context):
+    AccountConnectionWizard.selectAdvancedConfig()
+    AccountConnectionWizard.isLocalSyncDirVisible()
```

Comment:
Add the assertion in the Then step
```suggestion
@Then("local folder selection option should be visible")
def step(context):
    visible = AccountConnectionWizard.isLocalSyncDirVisible()
    test.compare(visible, True, <some-message>)
```